conduct/artifacts/CODE_OF_CONDUCT_PATCH.patch

```diff
@@ -29,29 +20,3 @@
  All complaints will be reviewed and investigated promptly and fairly.
  
  All community leaders are obligated to respect the privacy and security of the
-@@ -155,13 +157,13 @@
- individual, or aggression toward or disparagement of classes of individuals.
- 
- **Consequence**: A permanent ban from any sort of public interaction within the
--community.
-+project community.
- 
- ## Attribution
- 
- This Code of Conduct is adapted from the [Contributor Covenant][homepage],
--version 2.1, available at
--[https://www.contributor-covenant.org/version/2/1/code_of_conduct.html][v2.1].
-+version 2.0, available at
-+[https://www.contributor-covenant.org/version/2/0/code_of_conduct.html][v2.0].
```

Comment:
isn't the point of this PR to change from 2.0 to 2.1?

Reply:
This line is removed.